csrc/host_ir/jit.cpp

```diff
@@ -488,9 +490,10 @@ void inferTensorShapesAndStrides(
   }
 
   // Check if sizes and strides are the same size as logical domain
-  NVF_ERROR_EQ(sizes.size(), TensorDomain::noReductions(logical_domain).size());
-  NVF_ERROR_EQ(
-      strides.size(), TensorDomain::noReductions(logical_domain).size());
+  const auto logical_ndims =
```

Comment:
nitpick: Is it supposed to be this according to style guide? 
```suggestion
  const auto kLogicalNdims =
```
Although, I personally prefer to use snake-case regardless of const-ness.